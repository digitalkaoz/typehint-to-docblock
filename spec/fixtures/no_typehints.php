<?php

namespace Foo;

class Test
{
    public function it_does_things($foo, $cat)
    {
    }
}
