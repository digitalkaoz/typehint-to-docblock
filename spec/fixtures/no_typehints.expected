<?php

namespace Foo;

class Test
{
    public function does_things($foo, $cat)
    {
    }
}